easybuild/easyconfigs/h/h5py/h5py-2.8.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -14,20 +14,22 @@ toolchainopts = {'usempi': True}
 
 source_urls = [PYPI_SOURCE]
 sources = [SOURCE_TAR_GZ]
-checksums = ['e626c65a8587921ebc7fb8d31a49addfdd0b9a9aa96315ea484c09803337b955']
-
-# to really use mpi enabled hdf5 we now seem to need a configure step
-prebuildopts = ' python setup.py configure --mpi --hdf5=$EBROOTHDF5 && '
+patches = ['h5py-2.10.0_avoid-mpi-init.patch']
+checksums = [
+    'e626c65a8587921ebc7fb8d31a49addfdd0b9a9aa96315ea484c09803337b955',  # h5py-2.8.0.tar.gz
+    '6bacb71f5d9fbd7bd9a01018d7fe21b067a2317f33c4a7c21fde9cd404c1603f',  # h5py-2.10.0_avoid-mpi-init.patch
+]
 
 dependencies = [
     ('Python', '2.7.15'),
     ('HDF5', '1.10.2'),
     ('pkgconfig', '1.3.1', '-Python-%(pyver)s'),
 ]
 
-sanity_check_paths = {
-    'files': [],
-    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
-}
+use_pip = False
```

Comment:
Would you like to rather use pip with https://github.com/easybuilders/easybuild-easyconfigs/pull/11044 ?
I'd recommend doing that and also adding the `sanity_pip_check = True`